examples/2016-10-31/http_api_cors/README.md

```diff
@@ -0,0 +1,12 @@
+# HttpApi with CORS Example
+
+Example SAM template to configure CORS for HttpApi
+
+## Running the example
+
+```bash
+$ sam deploy \
+    --template-file /path_to_template/packaged-template.yaml \
```

Comment:
Can just do a local path to the template, as it doesn't need to be packaged (inline code doesn't require any packaging).